extensions/typescript-language-features/src/languageFeatures/organizeImports.ts

```diff
@@ -47,7 +48,8 @@ class OrganizeImportsCommand implements Command {
 					file
 				}
 			},
-			skipDestructiveCodeActions: sortOnly,
```

Comment:
Will this break clients that are on TS 4.8? For those we still would need to send `skipDestructiveCodeActions`, right?

Reply:
Oh, of course. Good catch.